functional-tests/step_definitions/custom_test_steps.js

```diff
@@ -0,0 +1,28 @@
+const methods_interactions = require('kakunin').methods.interactions;
```

Comment:
1. We rather use camelCase in Kakunin
2. I think, that it might be better to import is as 
```
const { methods } = require('kakunin')
```